config/settings.yml

```diff
@@ -1072,19 +1075,32 @@
   :custom_logo: false
   :events:
     :disk_usage_gt_percent: 80
+  :heartbeat_timeout: 2.minutes
+  :heartbeat_frequency: 30.seconds
   :host:
   :listening_port: '443'
   :mks_classid: 338095E4-1806-4BA3-AB51-38A3179200E9
   :mks_version: 2.1.0.0
   :monitor_poll: 5.seconds
   :name: EVM
+  :prefetch_max_per_worker: 10
+  :prefetch_max_per_worker_dequeue: 100
+  :prefetch_min_per_worker_dequeue: 10
+  :prefetch_stale_threshold: 30.seconds
```

Comment:
I agree, however, all of the settings should be reevaluated for name and purpose after this PR.  The purpose of this PR should be just to "unhide" hidden settings in order to remove hardcoding defaults in the app.